tests/steps/repository.py

```diff
@@ -234,6 +251,13 @@ def valid_offer(context, state=None, expiry_date=None, random_id=None):
         context.body = 'invalid data'
 
 
+
+@given(u'request body is an indirect offer for set "{set_id}"')
```

Comment:
indirect referred here to offers using "asset_selector" and "set" to reference the asset - the necessity to qualitfy the offer is due to the fact that in current tests we tend to generate sets as part of the offer. The terminology 'indirect' was introduced on initial stories with tests - I agree we can probably find a better name to qualify these constructs.